lsb_vsx/lsb_vsx.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from trunner.ctx import TestContext
+from trunner.dut import Dut
+from trunner.types import Status, TestResult
+
+
+def harness(dut: Dut, ctx: TestContext, result: TestResult) -> Optional[TestResult]:
+    tc_start = r".+?\|TP Start\r+\n"
+    msg_re = r".+\|(?P<msg_line>.*?)\r+\n"
+    result_re = r"(.+?)\|(?P<status>PASS|FAIL|UNRESOLVED|UNSUPPORTED|NOTINUSE|UNTESTED)\r+\n"
+    final_re = r"(.+?)\|TC End.+?\r+\n"
+
+    stats = {"FAIL": 0, "IGNORE": 0, "PASS": 0}
+    msg = ""
+    msg_lines = []
+    results = []
+    tc_num = 0
+    get_msg = False
+
+    dut.expect("Config End\r+\n")
+
+    while True:
+        idx = dut.expect([tc_start, final_re, result_re, msg_re], timeout=1000)
+        parsed = dut.match.groupdict()
+
+        if idx == 0:
+            get_msg = True
+        elif idx == 1:
+            break
+        elif idx == 2:
+            # Right now, not supported statuses treat as PASS
+            if parsed["status"] in ("UNRESOLVED", "UNSUPPORTED", "NOTINUSE", "UNTESTED"):
+                parsed["status"] = "PASS"
+
+            status = Status.from_str(parsed["status"])
+            tc_num += 1
+            subname = f"testcase {tc_num}"
+            if msg_lines:
+                msg += msg_lines[0]
+                for line in msg_lines[1:]:
+                    msg += "\n" + " " * 26 + line.lstrip()
```

Comment:
It should work, but it’s up to you
```suggestion
                    msg += "\n\t\t" + line.lstrip()
```

Reply:
I think trunner should be able to format msg with new lines.